netstandard/tools/HandlePackageFileConflicts.cs

```diff
@@ -155,6 +241,75 @@ private int GetPackageRank(ITaskItem item)
             return rank;
         }
 
+        /// <summary>
+        /// Get's the key to use for identifying reference conflicts
+        /// </summary>
+        /// <param name="item"></param>
+        /// <returns></returns>
+        private static string GetReferenceKey(ITaskItem item)
+        {
+            var aliases = item.GetMetadata("Aliases");
+
+            if (!String.IsNullOrEmpty(aliases))
+            {
+                // skip conflict detection for aliased assemblies
```

Comment:
Because an alias means that it will not conflict at compile time.  I could try to model all the alias scopes and check each one individually for conflicts but that became much too complicated and error prone.  What happens when one assembly defines multiple aliases and only one conflicts?  Rather than add a ton of code to handle aliases I just decided to scope them out.  We won't help remove conflicts that are aliased.